ssl/statem/extensions_clnt.c

```diff
@@ -498,6 +495,35 @@ int tls_construct_ctos_supported_versions(SSL *s, WPACKET *pkt, X509 *x,
     return 1;
 }
 
+/*
+ * Construct a psk_kex_modes extension. We only have two modes we know about
+ * at this stage, so we send both.
+ */
+int tls_construct_ctos_psk_kex_modes(SSL *s, WPACKET *pkt, X509 *x,
```

Comment:
ok.  it's in mem.c because of some compiler, but as you point out there are zillions of other places, so maybe mem.c is outdated?  shrug.  ok.